tests/acceptance/bootstrap/SharingNgContext.php

```diff
@@ -2119,4 +2119,22 @@ public function userGetsFederatedPermissionsListForFileOfTheSpaceUsingTheGraphAp
 			$this->getPermissionsList($user, $fileOrFolder, $space, $resource, $query)
 		);
 	}
+
+	/**
+	 * @Then the updated permissions role of the last share should be :permissionsRole
+	 *
+	 * @param string $permissionsRole
+	 *
+	 * @return void
+	 */
+	public function theUpdatedPermissionsRoleOfTheLastShareShouldBe(string $permissionsRole): void {
+		$responseBody = $this->featureContext->getJsonDecodedResponseBodyContent();
+		$actualRoleId = $responseBody->roles[0];
```

Comment:
```suggestion
		$actualRoleId = $this->featureContext->getJsonDecodedResponseBodyContent()->roles[0];
```
Can we do this directly in one line to reduce the line?

Reply:
yes we can